application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -147,10 +157,17 @@ private void rawTag(@NotNull SlashCommandEvent event) {
         if (tagSystem.handleIsUnknownTag(id, event)) {
             return;
         }
-
-        event.replyEmbeds(MessageUtils.generateEmbed(null,
-                MessageUtils.escapeMarkdown(tagSystem.getTag(id).orElseThrow()), event.getUser(),
-                TagSystem.AMBIENT_COLOR, null))
+        /*
+         * MessageUtils.generateEmbed(null,
+         * MessageUtils.escapeMarkdown(tagSystem.getTag(id).orElseThrow()), event.getUser(),
+         * TagSystem.AMBIENT_COLOR, null)
+         */
+        event
+            .replyEmbeds(new EmbedBuilder()
+                .setDescription(MessageUtils.escapeMarkdown(tagSystem.getTag(id).orElseThrow()))
+                .setAuthor(String.valueOf(event.getUser()))
```

Comment:
Why are you using `setAuthor` now? Thats wrong. Use `setFooter` like it was in the old code. Dont change behavior. Also, `String.valueOf(event.getUser())` is definitely wrong.

Reply:
> Why are you using `setAuthor` now? Thats wrong. Use `setFooter` like it was in the old code. Dont change behavior. Also, `String.valueOf(event.getUser())` is definitely wrong.

I was using setAuthor because I have replaced the MessageUtils generate embed method, and all the values where it has get user like in the image I have added an author there, and hence the comments also so you guys could see if I have modified some behaviors or added some extras 

![image](https://user-images.githubusercontent.com/58050604/145600365-68f386f0-3921-410d-b658-0b9ff58923e5.png)

Reply:
The old code used a `setFooter` though, not `setAuthor`.